applications/sandbox/systems/testsystem.cpp

```diff
@@ -1,9 +1,39 @@
 #include "testsystem.hpp"
+
+//#define LLRI_ENABLE_VALIDATION 0 //uncommenting this disables internal validation (see docs)
+//#define LLRI_ENABLE_INTERNAL_API_MESSAGE_POLLING 0 //uncommenting this disables internal API message polling
 #include <llri/llri.hpp>
 
+namespace llri = legion::graphics::llri;
+
+void callback(const llri::validation_callback_severity& severity, const llri::validation_callback_source& source, const char* message, void* userData)
+{
+    lgn::log::severity sev = lgn::log::severity_info;
+    switch (severity)
+    {
+        case llri::validation_callback_severity::Verbose:
+            return; //Comment out return and uncomment the following 2 lines to get verbose callbacks
+            //sev = lgn::log::severity_trace;
+            //break;
+        case llri::validation_callback_severity::Info:
+            return; //Comment out return and uncomment the following 2 lines to get info callbacks
+            //sev = lgn::log::severity_info;
+            //break;
```

Comment:
True, some validaiton info messages can be a bit excessive (especially in Vulkan where it lists a ton of information) but i guess I could set both to trace to avoid that flooding consoles